apps/router/src/components/ServiceMenu.tsx

```diff
@@ -70,12 +70,14 @@ export const ServiceMenu: React.FC<ServiceMenuProps> = ({
       closeOnBlur={true}
     >
       <MenuButton
-        as={Button}
-        leftIcon={<FiMenu />}
+        as={IconButton}
+        icon={<FiGrid />}
         size='md'
         variant='outline'
         onClick={() => setIsOpen(!isOpen)}
-      />
+      >
+        Services
```

Comment:
localize this string?

Reply:
Actually, this is an unnecessary string so I can just remove it. Thanks for pointing it out 👍